recipes/nglview/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: nglview
+  version: "0.5.dev"
+
+source:
+  git_url: https://github.com/arose/nglview
```

Comment:
@bgruening: so let this PR open until nglview release v0.5. The current v0.4 has a bug that causes travis failed. https://travis-ci.org/bioconda/bioconda-recipes/jobs/122976102#L1789